app/services/provider_authorisation.rb

```diff
@@ -10,7 +10,7 @@ def can_make_offer?(application_choice:, course_option_id:)
 
     course_option = CourseOption.find(course_option_id)
     training_provider = course_option.provider
-    ratifying_provider = course_option.course.accredited_provider
+    ratifying_provider = course_option.course.accredited_provider || training_provider
```

Comment:
Changing the roles around here seems surprising. Maybe we need to have explicit language about "self-ratifying" providers?

Reply:
Turns out, the logic is wrong. The data sync task will not create `ProviderRelationshipPermissions` records for self-ratified courses and we currently interpret the absence of records as lack of permissions, denying self-ratifying providers the ability to make decisions.

Discussing this with Duncan, we've come to the conclusion we should be skipping org-level permissions checks for self-ratified courses.